api/spec/v1.yaml

```diff
@@ -875,15 +907,30 @@ components:
     Transaction:
       type: object
       properties: 
-        height:
+        block:
           type: integer
           format: int64
           description: The block height at which this transaction was executed.
           example: *block_height_1
+        index_in_block:
```

Comment:
I guess we want to make these more frontend friendly?

Reply:
I thought this particular name/change `index_in_block` was to there to make it easier for frontend devs to consume. My soft preference is to stay as close to the backend as possible unless it introduces a lot of confusion.

Also, I guess this is resolved by going to `index`.